modules/local/custom/lilac_realign_reads_lilac/main.nf

```diff
@@ -0,0 +1,74 @@
+process CUSTOM_REALIGNREADS {
+    tag "${meta.id}"
+    label 'process_low'
```

Comment:
Since the reads for realignment includes only those related to MHC class I genes, the input is very small and the process takes <5 minutes even with `process_low` resources